assets/src/edit-story/app/story/actions/useSaveStory.js

```diff
@@ -25,18 +25,37 @@ import { addQueryArgs } from '@wordpress/url';
  */
 import { useAPI } from '../../api';
 import { OutputPage } from '../../../output';
+import { useConfig } from '../../config';
 
 /**
  * Creates AMP HTML markup for saving to DB for rendering in the FE.
  *
- * @param {Object} pages Object of pages.
+ * @param {Object} story Story object.
+ * @param {string} story.featuredMediaUrl Featured media URL.
```

Comment:
Why just this one? `story.title` is also used in the rendering.

Reply:
I stopped mid-way because it might be easier to just extract this to a `@typedef`